lib/redis_client/ruby_connection.rb

```diff
@@ -77,6 +77,8 @@ def initialize(config, connect_timeout:, read_timeout:, write_timeout:)
         read_timeout: read_timeout,
         write_timeout: write_timeout,
       )
+
+      @buffer = String.new(encoding: Encoding::BINARY, capacity: 127)
```

Comment:
That `127` is quite arbitrary, would be worth putting some thoughts into it, or making it configurable.

Reply:
This was taken from the buffer definition it already uses - https://github.com/redis-rb/redis-client/blob/aa5a3082faa83ecdd149d6d80df72259df128623/lib/redis_client/ruby_connection/resp3.rb#L53-L55